coverage_comment/storage.py

```diff
@@ -158,8 +160,16 @@ def get_file_url(
     # seconds.
 
 
-def get_readme_url(repository: str, branch: str) -> str:
-    return f"https://github.com/{repository}/tree/{branch}"
+def get_readme_url(repository: str, branch: str, path: str | None = None) -> str:
+    part = "blob" if path else "tree"
+    return f"https://github.com/{repository}/{part}/{branch}{path or ''}"
```

Comment:
I'm not sure I understand your comment. Without the slash you can always pass `/./coverage_comment`as path, no ?

Also, this changes the URL when no path is given, adding a trailing slash.

I'm not saying I'm against it, but I don't understand.

Reply:
My idea was simply to allow as wide a range of paths as possible, even if they don't start with a slash. But this is probably nonsense. We should just make sure to always put a slash at the beginning.

Reply:
Ah I get it.

Reply:
I think I'm going to apply your suggestion but right strip slashes on the final string

Reply:
:+1:

Reply:
Ok, I've refactored the function so that it uses trailing slash to determine if the path is a folder or a file, and the leading slash is ignored. Both empty string and single slash are accepted and understood as "root" (so, a folder)